Documentation/getting-started-meson-build.md

```diff
@@ -19,41 +22,48 @@ The following dependencies are needed to compile/build/run.
 * glib 2.0
 * meson >= 0.50
 * ninja-build
-* Neural network frameworks or libraries for plugins (e.g., tensorflow) you want to use, including their pkgconfig files or mechanisms to support meson to discover the corresponding frameworks. If you use our packages for tensorflow/pytorch, you do not need to worry.
-    * Possible frameworks for "extra" plugins: tensorflow, pytorch, protobuf, flatbuf, openvino, ncsdk2, Verisilicon-Vivante, SNPE, ...
+* Neural network frameworks or libraries for plugins (e.g., tensorflow) you want to use, including their pkgconfig files or mechanisms to allow meson to discover its headers and libraries. If you use development pacakges packaged by us for tensorflow/pytorch/openvino/..., you do not need to worry.
+    * Possible frameworks for "extra" plugins: tensorflow, tensorflow-lite, pytorch, protobuf, flatbuf, openvino, ncsdk2, Verisilicon-Vivante, SNPE, TensorRT, mqtt, ...
 * [SSAT](https://github.com/myungjoo/SSAT) (optional. required for unit testing)
 * gtest (optional. required for unit testing)
 
 The minimal requirement to build nnstreamer with default configuration is
 ```bash
-$ sudo apt install meson ninja-build gcc g++ libglib2.0-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
+$ sudo apt-get install meson ninja-build gcc g++ libglib2.0-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
+```
+
+Optional utilities:
+```bash
+$ sudo add-apt-repository ppa:nnstreamer/ppa
+$ sudo apt-get update
+$ sudo apt-get install libiniparser-dev=4.1 # if your inparser is < 4.1
+$ sudo apt-get install ssat # if you want to run unit tests
 ```
+You may build and install iniparser and ssat directly from their sources, too.
 
-***1. Clone the needed repositories***
+***1. Clone nnstreamer***
 
 ```bash
-$ git clone https://github.com/myungjoo/SSAT ssat
-$ git clone https://git.tizen.org/cgit/platform/upstream/tensorflow
 $ git clone https://github.com/nnstreamer/nnstreamer
 ```
 
-***2. Fix tensorflow build errors***
+***2. Install related frameworks***
 
-You may skip this if you have downloaded binary packages from PPA.
 
-A script, ```tensorflow/contrib/lite/Makefile```, may fail, depending your shell.
-Replace the ARCH macro (HOST_ARCH in some versions):
-
-From:
-```makefile
-ARCH := $(shell if [[ $(shell uname -m) =~ i[345678]86 ]]; then echo x86_32; else echo $(shell uname -m); fi)
+*Option A*: You may install optional developmental packages from nnstreamer PPA for more subplugins and extra plugins (choose what you need):
 ```
-
-To:
-```makefile
-ARCH := $(shell uname -m | sed -e 's/i[3-8]86/x86_32/')
+$ sudo add-apt-repository ppa:nnstreamer/ppa
+$ sudo apt-get update
+$ sudo apt-get install libedgetpu-dev libflatbuffers-dev libgrpc-dev openvino-dev libpaho-mqtt-dev libprotobuf-dev pytorch tensorflow-c-dev tensorflow-lite-dev tensorflow2-lite-dev tvm-runtime
```

Comment:
tvm-runtime > tvm-runtime-dev to build tvm subplugin

Reply:
Applied.